web/modules/custom/joinup_user/src/Routing/RouteSubscriber.php

```diff
@@ -19,6 +19,12 @@ protected function alterRoutes(RouteCollection $collection) {
         '_title_callback' => 'joinup_user_canonical_title',
       ]);
     }
+
+    if ($route = $collection->get('user.login')) {
+      $route->addDefaults([
+        '_title' => 'Sign in',
+      ]);
+    }
```

Comment:
Ahhh!! Here is the reason why the title of the login form was removed from the form alter :)